src/dsp_tools/utils/connection_live.py

```diff
@@ -34,11 +34,15 @@ class ConnectionLive:
 
     server: str
     token: Optional[str] = None
+    session: requests.Session = field(init=False, default=requests.Session())
```

Comment:
could we perhaps move this to an import? so that we write Session = .... 
it is less verbose and we can always check the imports if we want to know where it is from (we don't need that info on every line that object is called)